### PR TITLE
Adding repositories

### PR DESCRIPTION
**Pull request by CBID2**
---
## 📢 Only PRs from existing Virtual Coffee members will be accepted.
---

## Type of Contribution

Check the item(s) that applies.

- [ ] 🌱 Practice Open Source
- [X] 📃 Repositories List

### If your type of contribution is Repositories List, check items in the repository checklist below that apply.

⚠️ _We only accept PRs where most of the items are checked_

- [X] Relevant name
- [X] Description
- [X] Relevant tags
- [X] README.md file
- [X] CONTRIBUTIONS.md/CONTRIBUTING.md file
- [X] Open source license
- [X] Code of Conduct (COC)
- [X] Issue and Pull Request templates

## Description
This PR adds a couple of repos for Hacktoberfest

### CODE REVIEW

File: repositories-list.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ Templates for making issues and pull requests.
 - [Virtual Coffee / podcast transcripts](https://github.com/Virtual-Coffee/podcast-transcripts)
 - [freeCodeCamp's Developer Quiz Site](https://github.com/freeCodeCamp/Developer_Quiz_Site)
 - [EddieHubCommunity / BioDrop](https://github.com/EddieHubCommunity/BioDrop)
+- [LinksHub](https://github.com/rupali-codes/LinksHub)
+- [Codecademy/docs](https://github.com/Codecademy/docs)
+- [TheOdinProject/curriculum](https://github.com/TheOdinProject/curriculum)
+- [MDN Web Docs/content](https://github.com/mdn/content)
+- [OpenSauced/docs](https://github.com/open-sauced/docs)
```
Comment (adiati98) [inline repositories-list.md:33]: There's a duplication with line 36. Please remove this.

File: repositories-list.md [hunk 2]
```diff
@@ -26,6 +26,12 @@ Templates for making issues and pull requests.
...
```
Comment (adiati98) [inline repositories-list.md:33]: Hey @CBID2,
I'm not sure if you have removed the OpenSauced Docs repo here. Because there is a duplication in line 35.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Thanks for the change, @CBID2!
However, I still see a duplication here.

File: repositories-list.md (hunk 2 above)
Comment (CBID2) [reply repositories-list.md:33]: I removed the duplicate @adiati98.

**Pull request merged**